src/modules/tools/zprobe/ZProbe.cpp

```diff
@@ -362,8 +369,16 @@ void ZProbe::on_gcode_received(void *argument)
             return;
         }
 
+        if(gcode->subcode == 4 || gcode->subcode == 5) {
+            pin.set_inverting(pin.is_inverting() != 1);
```

Comment:
This is an ugly hack, sorry but not acceptable

Reply:
We just need a bit to say invert the test and xor that in the read_probe function.